documentation/onboard-dpg-in-sdkautomation/README.md

```diff
@@ -0,0 +1,37 @@
+# Service Onboard DPG with Swagger CI Pipeline
+
+Service Onboarding DPG with Swagger CI pipeline can help you get generated SDK and ApiView during the procedure of creating swagger PR.
+The benefits are following:
+1. API signature could be reviewed at the same time when we submit a Swagger PR. This could help to detect API issues at the early stage.
+2. It enable service teams to access their SDKs even before their Swagger PR is merged.
+
+This Document describes the process of how to onboard DPG with SDK automation pipeline. __This document is targeting for service team, who wants to onboard DPG.__ 
+
+*If you want to go through details of pipeline implementation, please go to [Integrate DPG and ApiView in SDK Automation Pipeline](Integrate-dpg-and-apiview-in-sdk-automation-pipeline.md).*
+
+## WorkFlow
+
+![workflow-for-service-team](workflow-service-team.png)
+
+__Description:__
+1. Creates a swagger PR with [Guide to design and creation of Data Plane REST API and Client Libraries](https://aka.ms/azsdk/dpcodegen).
+2. If the SDK was generated for this service before, and you don't need to change the autorest configuration in sdk repository, please go to step 4. Otherwise, please go to step 3.
+3. Create branch with name `swagger/<prNumber>` in sdk repository, and add/update autorest configuration file in the branch. Then add a comment `/azp run` in swagger PR.
```

Comment:
For the branch name we try to come up with something a little more specific then `swagger/<prnumber>` as we also need to include branch clean-up policies for these. Perhaps a variation of the existing `sdkauto-onboarding/<prnumber>` or something like that.